README.md

```diff
@@ -28,14 +27,14 @@ inside container, via the [kas-container] script.
 
 * [kas-container] installation:
 
-```
+```sh
 wget -O ~/bin/kas-container https://raw.githubusercontent.com/siemens/kas/master/kas-container
```

Comment:
You should probably use permalink to the version you want e.g.
https://github.com/siemens/kas/blob/422e6032f6a86aa7a3d47671e31f08650e5bd228/kas-container
or
https://raw.githubusercontent.com/siemens/kas/4.4/kas-container
if you want it to be more readable, but tags can be changed so safer would be to use commit hash

Reply:
[Adressed](https://github.com/3mdeb/meta-rte/pull/73/commits/a21267c9321973e42d181b8fe1e8ab375e60a782)